mlir/lib/Dialect/Utils/IndexingUtils.cpp

```diff
@@ -213,6 +213,14 @@ mlir::invertPermutationVector(ArrayRef<int64_t> permutation) {
   return inversion;
 }
 
+bool mlir::isIdentityPermutation(ArrayRef<int64_t> permutation) {
+  int n = permutation.size();
```

Comment:
Let's use int64_t to keep the type consistent with the element types in permutation.

Reply:
I'm changing this to `for (auto i : llvm::seq<int64_t>(0, permutation.size()))` so we won't need `n`. And `i` uses int64_t type.

Reply:
sounds good to me, thanks!